synchronization/src/cyface/java/de/cyface/synchronization/CyfaceSyncAdapter.java

```diff
@@ -233,23 +236,53 @@ public void onPerformSync(final @NonNull Account account, Bundle extras, String
                 }
             } while (unsyncedMeasurementsCursor.moveToNext());
 
-        } catch (MalformedURLException e) {
+        }
+        // We currently inform two groups of listeners: the error intent listeners (to show the
+        // error) and the syncProgress listeners to upgrade the sync progress UI
```

Comment:
Why isn't the syncProgress listener listening to the error handler as well? In that case it would not need to do its own error handling, wouldn't it?

Reply:
Habe einen Task erstellt, wenn ich alles in einem großen Aufwasch mache, werde ich nicht fertig bzw. der PR zu rießig.